tests/UI/utils/helpers.ts

```diff
@@ -61,6 +61,8 @@ export default {
             width: global.BROWSER.width,
             height: global.BROWSER.height,
           },
+        // @todo : Remove it when Puppeteer will accept self signed certificates
+        ignoreHTTPSErrors: false,
```

Comment:
Do you need it with the value `false` ?

Reply:
Yes, for the moment, i will remove it in a future PR.

Reply:
It's counter intuitive, I read "do not ignore https errors" with the comment about self signed certificates but ok :D